NSS-Day-Cohort-49/giffygram-ermagerds
# Mat login

#### Changes Made
1. Modified login screen to dispatch events to re-render html
2. modified main to fetch data before rendering
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > when page loads and local storage is empty, user should see login screen
    > when valid user information is input and button clicked, should re-render, showing home page
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
